addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -42,6 +42,9 @@ var MassMailingFieldHtml = FieldHtml.extend({
         var self = this;
         var fieldName = this.nodeOptions['inline-field'];
 
+        if (!this.wysiwyg) {
```

Comment:
Instead of check instance of wysiwyg, we should check readonly mode of widget for code flow readability like
if (this.mode == "readonly) return super;

cc @Gorash

Reply:
:+1:

Reply:
@Gorash fix has been applied as suggested. Also test case has been added to this pr. Thanks!